frontend/src/components/hub/HubContent.js

```diff
@@ -133,7 +135,13 @@ export default function HubContent({
             <div className={`${user && classes.dashboardAndStatboxWrapper}`}>
               <div>
                 {user ? (
-                  <Dashboard location={location} headline={headline} hubUrl={hubUrl} />
+                  <Dashboard
+                    allHubs={allHubs}
+                    hubData={hubData}
+                    location={location}
+                    headline={headline}
+                    hubUrl={hubUrl}
```

Comment:
Nit: do you mind alphabetizing these? If you're using VS Code, this is really easy by pressing `cmd+shift+p` and then typing "sort".

Reply:
Sure, no problem. Personally, I don't alphabetize shortcuts but if it is our standard, I will definitely follow it. :-)

Reply:
Hmmm I've checked it in code and we have "hubUrl" everywhere. Do we really want to do alphabetize it?

Reply:
I feel like doing this manually doesn't make too much sense at this point since we've already not been doing it in the rest of the codebase. Instead we should probably update our configuration for `prettier` so this gets done automatically.